Integrations/GitHub/GitHub.yml

```diff
@@ -0,0 +1,438 @@
+      description: Assignees
+      type: Unknown
+    - contextPath: GitHub.Issue.Labels
+      description: Labels
+      type: Unknown
+  - arguments:
+    - default: false
+      description: Title of the Issue
+      isArray: false
+      name: title
+      required: true
+      secret: false
+    - default: false
+      description: Issue body
+      isArray: false
+      name: body
+      required: false
+      secret: false
+    - default: false
+      description: Assignees
+      isArray: true
+      name: assignees
+      required: false
+      secret: false
+    - default: false
+      description: labels
+      isArray: true
+      name: labels
+      required: false
+      secret: false
+    deprecated: false
+    description: Creates a new issue
+    execution: false
+    name: create-issue
+    outputs:
+    - contextPath: GitHub.Issue.ID
+      description: Issue number
+      type: String
+    - contextPath: GitHub.Issue.Title
+      description: Issue title
+      type: String
+    - contextPath: GitHub.Issue.body
+      description: Issue body
+      type: String
+    - contextPath: GitHub.Issue.Assignees
+      description: Assignees
+      type: String
+    - contextPath: GitHub.Issue.Labels
+      description: Labels
+      type: String
+  - arguments:
+    - default: false
+      description: Issue number
+      isArray: true
+      name: issue_number
+      required: true
+      secret: false
+    deprecated: false
+    description: Closes issue
+    execution: false
+    name: close-issue
+  - arguments:
+    - default: false
+      description: Issue number
+      isArray: false
+      name: issue_number
+      required: true
+      secret: false
+    - default: false
+      description: Issue title
+      isArray: false
+      name: title
+      required: false
+      secret: false
+    - default: false
+      description: Issue body
+      isArray: false
+      name: body
+      required: false
+      secret: false
+    - default: false
+      description: Assignees
+      isArray: false
+      name: assignees
+      required: false
+      secret: false
+    - default: false
+      description: Labels
+      isArray: false
+      name: labels
+      required: false
+      secret: false
+    deprecated: false
+    description: Updates an issue
+    execution: false
+    name: update-issue
+    outputs:
+    - contextPath: GitHub.Issue.ID
+      description: ID
+      type: String
+    - contextPath: GitHub.Issue.Title
+      description: Title
+      type: String
+    - contextPath: GitHub.Issue.Body
+      description: Body
+      type: String
+    - contextPath: GitHub.Issue.Assignees
+      description: Assignees
+      type: String
+    - contextPath: GitHub.Issue.Labels
+      description: Labels
+      type: String
+  - deprecated: false
+    description: Get download count for all releases
+    execution: false
+    name: download-count
+  - arguments:
+    - default: false
+      description: Query
```

Comment:
this is a complex value. description should contain more information and an example